src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/SolutionExplorerInProcess.cs

```diff
@@ -566,8 +552,17 @@ private async Task<string> WaitForBuildToFinishAsync(IVsOutputWindowPane buildOu
                 return string.Empty;
             }
 
-            // The build summary line should be second to last in the output window
-            return lines[^2].Extent.GetText();
+            // Find the build summary line
+            for (var index = lines.Count - 1; index >= 0; index--)
+            {
+                var lineText = lines[index].Extent.GetText();
+                if (lineText.StartsWith("========== Build:"))
```

Comment:
```suggestion
                if (lineText.StartsWith("=========="))
```

Since that won't depend on localization anymore.

Reply:
I agree, but the reality is that every test that uses this method immediately Asserts the returned string equals an English build summary. Probably long term we should capture the success, failed, and canceled values returned by the UpdateSolutionEvents Done and return those instead of the build summary.